packetbeat/_meta/kibana/6/dashboard/Packetbeat-tls.json

```diff
@@ -429,8 +429,8 @@
                         }
                     }
                 }, 
-                "savedSearchId": "bf3d23b0-d37c-11e7-9914-4982455b3063", 
-                "title": "TLS Versions", 
+                "savedSearchId": "bf3d23b0-d37c-11e7-9914-4982455b3063-7x", 
+                "title": "TLS Version 7.xs 7.x", 
```

Comment:
This one does seem to be an artifact of a previous run, however. I don't see this in [my resulting file](https://github.com/webmat/beats/blob/find-replace-dashboards-mats-try/packetbeat/_meta/kibana/6/dashboard/Packetbeat-tls.json#L433)